src/result/sfProvarAutomationTestRunResult.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  ErrorHandler,
+  GenericErrorHandler,
+  SfProvarCommandResult,
+  SFProvarResult,
+} from '@provartesting/provardx-plugins-utils';
+import { Messages } from '@salesforce/core';
+
+/* eslint-disable */
+export class SfProvarAutomationTestRun extends SFProvarResult {
```

Comment:
class name and file name are different. Also the name seems misleading for a class which has a function related to populating result.